tbx/static_src/javascript/components/desktop-sub-menu.js

```diff
@@ -51,6 +54,15 @@ class DesktopSubMenu {
                 this.body.classList.add('no-scroll');
             }
         });
+
+        // Close the desktop menu when the focus moves away from the last item
+        this.lastMenuItems.forEach((item) => {
+            item.addEventListener('focusout', () => {
+                this.toggleNode.classList.remove('active');
+                this.node.setAttribute('aria-expanded', 'false');
+                this.body.classList.remove('no-scroll');
```

Comment:
Suggestion: As this is repeats on lines 40-42 could move it to it's own function - not a biggie though.

Reply:
done.